File: swagger/models/update_appointment_response.py
from .base_model import BaseModel

from .appointment import Appointment

class UpdateAppointmentResponse(BaseModel):
	"""
	no description available

	Attributes:

		appointment
		Contains information about the appointment.
	"""

	input_map = {
		'Appointment': ('appointment', Appointment),
		}

	output_map = {
		'appointment': 'Appointment',
		}

		'Appointment': ('appointment', Appointment),

File: swagger/Models/UpdateAppointmentResponse.php
<?php

namespace App\Mindbody\Models;

/**
 * no description available
 */
class UpdateAppointmentResponse extends BaseModel
{
	/**
	 * Contains information about the appointment.
	 * 
	 * @var Appointment 
	 */
	public $appointment;

	protected function getInputMap()
	{
		return [
			'Appointment' => ['appointment', Appointment::class, null],
			];
	}

	protected function getOutputMap()
	{
		return [
			'appointment' => ['Appointment', Appointment::class, null],
			];
	}
}
